app/helpers/application_helper.rb

```diff
@@ -389,8 +389,12 @@ def bibdata_location_code_to_sym(value)
   end
 
   def render_location_code(value)
-    location = Bibdata.holding_locations[value.to_sym]
-    location.nil? ? value : "#{value}: #{location_full_display(location)}"
+    if value.is_a?(Array)
+      value.map { |v| render_location_code(v) }
```

Comment:
@sandbergja Recursion is a bit complex in this situation.  I might have just cast the single value to an Array with 
```
Array(value).map do |loc|
   location = Bibdata.holding_locations[loc.to_sym]
   location.nil? ? value : "#{value}: #{location_full_display(location)}"
end
```